runtime/packages/modules/aura/assembly/aura.ts

```diff
@@ -34,6 +34,12 @@ export class AuraModule {
     static setAuthorities(auths: u8[]): void {
         Storage.set(AuraModule.AURA_AUTHORITIES, auths);
     }
+
+    static createInherent(data: InherentData): u8[] {
```

Comment:
Just FYI -> I think that it would be better the `createInherent` function to follow some kind of `interface` and return inherent like the timestamp module does.